lib/widgets-customize.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/**
+ * The sanitization function for incoming values for the `gutenberg_widget_blocks` setting.
+ * It's a JSON string, so it decodes it and encodes it again to make sure it's valid.
+ *
+ * @param string $value The incoming value.
+ */
+function gutenberg_customize_sanitize( $value ) {
+	return json_encode( json_decode( $value ) );
```

Comment:
Hm not sure this is right. If `$value` contains invalid JSON then this would return the string `"null"`.

```
php > var_dump(json_encode(json_decode("{")));
string(4) "null"
```